rusty_results/tests/option/test_option.py

```diff
@@ -32,3 +32,11 @@ def test_option_contains():
 def test_option_iter():
     assert list(iter(Empty())) == []
     assert list(iter(Some(1))) == [1]
+
+
+def test_from_optional():
+    assert Some(0) == OptionProtocol.from_optional(0)
```

Comment:
I think we should use the type alias `Option` instead of the protocol for this calls. They are a bad example 😅

Reply:
Updated to `option_from` function.